library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/Section.java

```diff
@@ -39,11 +40,13 @@
      * @param itemResourceId layout resource for its items
      * @param loadingResourceId layout resource for its loading state
      * @param failedResourceId layout resource for its failed state
+     * @param emptyResourceId layout resource for its empty state
      */
-    public Section(int itemResourceId, int loadingResourceId, int failedResourceId) {
```

Comment:
I agree with that. During my use case, I also take some time to debug what's went wrong, after I made such changes.

Perhaps, we can introduce Builder pattern for such case?